mutatest/run.py

```diff
@@ -281,91 +283,80 @@ def trial_output_check_break(
     Returns:
         Bool flag for whether or not to break the outer operations loop.
     """
-    if trial_results.status == "SURVIVED":
-        LOGGER.info(
-            "%s",
-            colorize_output(
-                (
-                    f"Surviving mutation at "
-                    f"{sample_src}: ({sample_idx.lineno}, {sample_idx.col_offset})"
-                ),
-                "red",
-            ),
-        )
-        if config.break_on_survival:
-            LOGGER.info(
-                "%s",
-                colorize_output(
-                    "Break on survival: stopping further mutations at location.", "red"
-                ),
-            )
-            return True
 
-    if trial_results.status == "DETECTED":
-        LOGGER.info(
-            "%s",
-            colorize_output(
-                (
-                    f"Detected mutation at "
-                    f"{sample_src}: ({sample_idx.lineno}, {sample_idx.col_offset})"
-                ),
-                "green",
-            ),
-        )
-        if config.break_on_detected:
+    @dataclass
+    class SwitchDatum:
+        status: str
+        output_desc: str
+        break_config_attr: str
+        break_desc: str
```

Comment:
Suggestion, `break_desc` could be a class property calculated from `break_config_attr` (replace underscores with spaces, capitalize first letter of first word) to ensure there is no accidental decoupling of the attr and the desc.

Similarly, `output_desc` could be a class property calculated from `status` - we can change the language in the descriptions to be consistent e.g., "Result: X at ..."